examples/gslib/compare.cpp

```diff
@@ -0,0 +1,227 @@
+﻿//          Serial example of utilizing GSLib's FindPoints methods
```

Comment:
I like the functionality of this very much. Please add documentation in the style of the miniapps.

Reply:
Can you please address this?